.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ env:
   RUSTFLAGS: -Dwarnings
   RUST_BACKTRACE: 1
   nightly: nightly-2020-09-21
-  minrust: 1.45.2
+  minrust: 1.47.0
```

Comment:
We can also bump only the version that runs clippy, because we're sticking to pin the version that runs clippy, not sticking to run clippy on the MSRV.